### DISCUSSION TITLE
`BigQueryIOManager` append or replace

### DISCUSSION THREAD

**Discussion by bendnorman** [dagster-io/dagster#24260] [Q&A] [answered]
Hello! Does the `BigQueryIOManager` append data to BQ tables or replace it? Is it possible to configure it?

**Comment by garethbrickman** [comment]
It doesn't look like this is documented as being possible for [BigQueryIOManager](https://docs.dagster.io/_apidocs/libraries/dagster-gcp#bigquery-i-o-manager) however see that [PolarsBigQueryIOManager](https://docs.dagster.io/_apidocs/libraries/dagster-polars#dagster_polars.PolarsBigQueryIOManager) has: 

>The “write_disposition” metadata key can be used to set the write_disposition parameter of bigquery.JobConfig. For example, set it to “WRITE_APPEND” to append to an existing table instead of overwriting it.

**Comment by sryza** [comment, marked as answer]
Currently, it always replaces. If the asset is partitioned, it will only replace the data in that particular partition (or "append" that partition if it has no exiting data).
